### PR TITLE
Created Nanoplot_tutorial.md

### PR DESCRIPTION
**Pull request by Thomieh73**
Added the command and text to the tutorial given on 20211025.

Karin, can you check this?

### CODE REVIEW

File: docs/source/tutorials/Nanopore_tutorial.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
+# Nanopore sequence data tutorial
+
+This is a tutorial to do quality control of the Nanopore sequence data. The data we use at our institute is mainly minION sequence data gnerate for the assembly of bacterial genomes. This tutorial uses Guppy version 5 for basecalling, demultiplexing and quality score filtering.  The tutorial is run a HIGH PERFORMANCE COMPUTING system that uses a SLURM system for job allocation. If you have your own server, just skip the srun step, but make sure to have Guppy, Nanoplot and NanoFilt installed.
```
Comment (karinlag) [inline docs/source/tutorials/Nanopore_tutorial.md:3]: data _generated_ for the

File: docs/source/tutorials/Nanopore_tutorial.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+| ----- | ----- |
+| Flowcell ID | FLO-MIN106  |
+| Sequencing kit ID | SQK-RBK004 |
+| Barcoding KIT ID |  RBK004    |
+
+### running Guppy basecalling
+
+Let's try and see if we can run Guppy, without blocking each other access to the GPU nodes.
+
+first let's active a `screen`. We call this screen : NANOPORE, so it is easy to find back if our connection is gone, and we have to log on to the system again. It is also easy to use when you open new screens all the time.
```
Comment (karinlag) [inline docs/source/tutorials/Nanopore_tutorial.md:54]: _First_

File: docs/source/tutorials/Nanopore_tutorial.md [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+```
+--disable_qscore_filtering : Disable filtering of reads into PASS/FAIL folders based on min qscore
+--min_qscore : Minimum acceptable qscore for a read to be filtered into the PASS folder
+--barcode_kits : Space separated list of barcoding kit(s) or expansion kit(s) to detect against. Must be in double quotes.
+--trim_barcodes : Trim the barcodes from the output sequences in the FastQ files.
+--num_barcode_threads: Number of worker threads to use for barcoding
+--disable_pings: stop ping the nanopore server.
+```
+
+So let us add the demultiplexing and reduce minimun quality score
```
Comment (karinlag) [inline docs/source/tutorials/Nanopore_tutorial.md:118]: _minimum_

File: docs/source/tutorials/Nanopore_tutorial.md [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+cd fastq
+```
+Let's run NanoPlot on the file `sequencing_summary.txt`
+
+```
+NanoPlot --summary sequencing_summary.txt --loglength -o summary-plots-log-transformed
+```
+That generates a folder called:  "summary-plots-log-transformed"
+
+let's take a look at the files. If you are on a cluster, you should download the folder to your own personal computer.
```
Comment (karinlag) [inline docs/source/tutorials/Nanopore_tutorial.md:155]: Let's

**Review by karinlag — COMMENTED** [review]
A few spelling mistakes otherwise good. Merge when that is done :)

File: docs/source/tutorials/Nanopore_tutorial.md (hunk 1 above)
Comment (Thomieh73) [reply docs/source/tutorials/Nanopore_tutorial.md:3]: done

File: docs/source/tutorials/Nanopore_tutorial.md (hunk 2 above)
Comment (Thomieh73) [reply docs/source/tutorials/Nanopore_tutorial.md:54]: done

File: docs/source/tutorials/Nanopore_tutorial.md (hunk 3 above)
Comment (Thomieh73) [reply docs/source/tutorials/Nanopore_tutorial.md:118]: done

File: docs/source/tutorials/Nanopore_tutorial.md (hunk 4 above)
Comment (Thomieh73) [reply docs/source/tutorials/Nanopore_tutorial.md:155]: done

**Review by Thomieh73 — COMMENTED** [review]
I have gone through the suggested changes, and resolved them.

**Comment by Thomieh73** [comment]
Merged the corrected documents with the repo

**Pull request merged**